Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -61,153 +51,108 @@ public static string FindRichestPerson(string[] peopleAndBalances)
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
             if (IsNullOrEmpty(peopleAndBalances)) return Constants.Nothing;
-            Thread.CurrentThread.CurrentCulture = Config.Culture();
-
-            var (names, money, isPlural) = ReturnNamesMoneyAndIsPlural(peopleAndBalances, "poor");
+            var balances = FindPeopleBalancesByOption(peopleAndBalances, "poor");
             var dictionary = new string[]
             {
-                (isPlural) ? "have" : "has",        // [0]
+                (balances[1] == "1") ? "have" : "has",
             };
 
-            return $"{names} {dictionary[0]} the least money. {FixMoneyFormat(money)}.";
+            return $"{balances[0]} {dictionary[0]} the least money. {balances[2]}.";
```

Comment:
Index of an array is a magic number. Use constants to name the index based on representing column.

Reply:
You say something like this?
```c#
var dictionary = new string[2];
dictionary[0] = (balances[1] == "1") ? "are" : "is";
dictionary[1] = (balances[1] == "1") ? "people" : "person";
```